packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1098,7 +1108,6 @@ export default class TradeStore extends BaseStore {
             return;
         }
         this.disposePreSwitchAccount();
-        this.disposeSwitchAccount();
```

Comment:
Why are we removing this line?

Reply:
hi @balakrishna-binary committed a wrong file. i will convert the PR to draft.